spec/swagger.yaml

```diff
@@ -5785,6 +5807,454 @@ paths:
+        items:
+          type: string
+        collectionFormat: multi
+        description: |
+          This value declares locales list which should be used when file is attached to a job.
+
+          **Note**: File will not be authorized for these locales once it is uploaded. Authorization is happened only when job is authorized.
+      - name: smartling.namespace
+        in: formData
+        required: false
+        type: string
+        description: |
+          This is only for accounts created after July 31st, 2015. Define a custom [namespace](https://developer.smartling.com/v2.0/docs/string-sharing-namespaces) for the file.
+
+          **Note**: While this parameter shares the format of other file directives, it will not work as an inline directive and must be specified in the API call.
+      - name: smartling.file_charset
+        in: formData
+        required: false
+        type: string
+        description: |
+          Specifies a custom charset for text-format files. The value should be the name of the character set. See a full list of supported character sets here.
+
+          If this directive is not used, Smartling will use the ```Content-Type``` request header to determine if the content is encoded with UTF-16, UTF-16B or UTF-16LE. If there is no ```Content-Type``` header, Smartling will examine the file for UTF-16 characters. If none are detected, UTF-8 encoding will be used.
+
+          **Note**: Once this property has been set for a file, it cannot be changed. If you reupload the file, it will use the original charset, even if you change the directive. An error will be returned if this directive is used when uploading binary-format file types, such as Office or IDML files.
+      - name: smartling.[command]
+        in: formData
+        required: false
+        type: string
+        description: |
+          Provides custom parser configuration for supported file types. See [Supported File Types](https://developer.smartling.com/docs/supported-file-types) for more details.
+      - name: callbackUrl
+        in: formData
+        required: false
+        type: string
+        format: url
+        description: |
+          A GET request that creates a callback to a URL when a file is 100% published for a locale. The callback gives the fileUri and locale with the format `http[/s]://your.url?locale=xx-XX&fileUri=your.file`. If you upload the file again, without a callbackUrl, it will remove any previous callbackUrl for that file. The RequestBin (http://requestb.in) service is a convenient way to test callback. RequestBin generates a short-lived disposable URL that displays all posted requests.
+
+          If a callback fails, Smartling will make multiple attempts to reach the designated URL, however, we do not keep retrying a failed callback indefinitely. Try to make sure your callback URL is as stable as possible and be aware that callbacks are not a 100% reliable way of being notified that a file is published.
```

Comment:
i think this should be rephrased. It comes across as we don't always send them.

Reply:
Our file-upload method just proxies the same one from Files API
I didn't find a way to refer to existing parameters descriptions so I just copypasted the doc from there (except two parameters - `authorize` and `localeIdsToAuthorize`) 
So I prefer to leave this consistent with the original doc